app/src/main/assets/json/onboarding.json

```diff
@@ -18,6 +18,62 @@
           "type"         : "permissions"
         }
       ]
+    },
+    {
+      "onboardingType" : "completion",
+      "steps": [
+        {
+          "identifier": "testerOrParticipant",
+          "title": "I am a:",
+          "type": "dataGroups.singleChoiceText",
+          "items": [
+            {
+              "text": "Tester",
+              "value": "test_user"
+            },
+            {
+              "text": "Participant",
+              "value": ""
+            }
+          ],
+          "expectedAnswer": "test_user",
+          "skipIdentifier": "clinicChoices",
+          "optional": false
+        },
+        {
+          "identifier": "testerChoices",
+          "title": "How do you want to experience the app while testing?",
+          "type": "dataGroups.singleChoiceText",
+          "items": [
+            {
+              "text": "Activities always available",
+              "value": "activity_tester"
+            },
+            {
+              "text": "As a participant would see it",
+              "value": "ux_tester"
+            }
+          ],
+          "optional": false
+        },
+        {
+          "identifier": "clinicChoices",
+          "title": "In which order are you doing the clinic visits?",
+          "type": "dataGroups.singleChoiceText",
+          "items": [
+            {
+              "text": "Cardiorespiratory stress test first visit, step and 12 minute run last visit",
+              "value": "clinic1"
+            },
+            {
+              "text": "Step and 12 minute run first visit, cardiorespiratory stress test last visit",
+              "value": "clinic2"
+            }
+          ],
+          "optional": false,
+          "shouldPersist": true
```

Comment:
So is the idea to have only "shouldPersist" on the last step?  That way it only needs to make one web call?